src/store/wizardSlice.ts

```diff
@@ -80,6 +84,9 @@ const initialState: wizardState = {
   openScap: {
     profile: undefined,
   },
+  kernel: {
+    kernelAppend: '',
+  },
```

Comment:
This is mostly a question - do we want to include kernel under the openScap state so we're consistent with storing values under a specific step? So like this?
```typescript
  openScap: {
    profile: undefined,
    kernel: {
      kernelAppend: '',
    },
  },
```

or is it its own thing? I might not be up to date with the compliance work so not sure what would the implications be.

Reply:
not sure, @lucasgarfield what is your opinion regarding that?

Reply:
Eventually there will be a step where the kernel arguments can be provided by the user, but we don't know whether it will be grouped with other options (for instance under an `Advanced` step) or by itself...

**For now let's put it under the openScap step as @regexowl suggests.** 

This is one of the nice things about using selectors to access everything from the state, we can easily change where it is stored in the state later and we'll only need to update the selector function.

Reply:
at the backend side its look sepeare from OpenScap 
<img width="596" alt="Screenshot 2024-02-08 at 14 56 17" src="https://github.com/osbuild/image-builder-frontend/assets/73419853/a5d58a5d-4330-444d-b85f-e59252224461">

are you sure you want it under OpenScap?